department-of-veterans-affairs/caseflow
# Judges can view AMA cases to assign and assign/reassign them between attorneys

Resolves https://github.com/department-of-veterans-affairs/caseflow/issues/6759


![image](https://user-images.githubusercontent.com/3781835/45443787-cbc60480-b693-11e8-89ec-17bd066977ea.png)
![image](https://user-images.githubusercontent.com/3781835/45443800-d2ed1280-b693-11e8-84b7-2d062bb806f0.png)

### Testing Plan
- Navigate to the "Assign Cases" view
- Verify that only unassigned cases show up in the judge tab
- Assign an AMA appeal to an attorney
- Reassign a mixture of AMA / legacy appeals to other attorneys


It may be good to prioritize https://github.com/department-of-veterans-affairs/caseflow/issues/6930 after this is merged.
